examples/plutus-nft/src/Cardano/PlutusExample/NFT.hs

```diff
@@ -104,37 +103,37 @@ $(deriveJSON defaultOptions ''MintParams)
 
 mintNft :: MintParams -> Contract (Last Text) NFTSchema Text ()
 mintNft MintParams {..} = do
-  pkh <- Contract.ownPaymentPubKeyHash
+  pkh <- Contract.ownFirstPaymentPubKeyHash
   utxos <- utxosAt (pubKeyHashAddress pkh Nothing)
   case Map.keys utxos of
     [] -> Contract.logError @Hask.String "no utxo found"
     oref : _ -> do
       tell $ Last $ Just $ "Using oref:" Hask.<> Text.pack (Hask.show oref)
       let cs = curSymbol oref mpTokenName
           val = singleton cs mpTokenName 1
-          meta =
-            NftMetadata $
-              Map.singleton cs $
-                Map.singleton mpTokenName $
-                  NftMetadataToken
-                    { nmtName = mpName
-                    , nmtImage = mpImage
-                    , nmtMediaType = Hask.pure "image/png"
-                    , nmtDescription = mpDescription
-                    , nmtFiles = Hask.mempty
-                    , nmtOtherFields = Hask.mempty
-                    }
+          -- meta =
```

Comment:
Should we keep this commented?

Reply:
same: https://github.com/mlabs-haskell/bot-plutus-interface/pull/103#discussion_r955396631